src/App.tsx

```diff
@@ -1,14 +1,55 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import './components/App.scss';
 
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoaded, setIsLoaded] = useState(false);
+  const [query, setQuery] = useState('');
+  const [selectFilter, setSelectFilter] = useState('all');
```

Comment:
Create an enum with all your filters
```suggestion
  const [selectFilter, setSelectFilter] = useState(FilterBy.All);
```